docs/index.rst

```diff
@@ -205,6 +205,70 @@ Topic Management
       .. figure:: images/create_topic.png
          :alt: image
 
+7. When Confluent Monitoring Interceptors are configured on any Kafka-based clients, i.e., producers, consumers, Connect, KSQL, then they write metadata to a topic called ``_confluent-monitoring``.
+   Confluent Control Center reads that topic to do expert stream monitoring for message delivery assurance and performance for throughput and latency.  
```

Comment:
suggest using text-string replace for C3: `|c3|` or the abbreviated `|c3-short|`

Reply:
I would change `i.e.` to `e.g.` and drop the trailing comma.

Reply:
Ack